grails-app/domain/org/pih/warehouse/order/OrderItem.groovy

```diff
@@ -285,14 +286,23 @@ class OrderItem implements Serializable, Comparable<OrderItem> {
         return quantityInvoicedInStandardUom >= quantity
     }
 
+    Integer getInvoicedQuantity() {
+        return allInvoiceItems
+                ?.findAll { !it?.invoice?.isPrepaymentInvoice }
+                ?.sum { it.quantity } ?: 0
```

Comment:
Could you add a comment that this quantity is already in standard uom? Here or in the `getQuantityAvailableToInvoice`, so it is clear?